src/main/kotlin/org/jetbrains/research/kotlin/mpp/inference/operators/OperatorFactory.kt

```diff
@@ -0,0 +1,27 @@
+package org.jetbrains.research.kotlin.mpp.inference.operators
+
+import TensorProto.DataType
+import org.jetbrains.research.kotlin.mpp.inference.operators.activations.Activation
+import org.jetbrains.research.kotlin.mpp.inference.operators.math.Add
+import org.jetbrains.research.kotlin.mpp.inference.operators.math.MatMul
+import org.jetbrains.research.kotlin.mpp.inference.types.resolveKClass
+
+@Suppress("UNCHECKED_CAST")
```

Comment:
Where is UNCHECKED_CAST here? Please, check all suppressions :)

Reply:
Reported cast is [here](https://github.com/JetBrains-Research/kotlin-mpp-inference/blob/e3adf13bf8f2a77f89d6c5c40cb44ba0cabea800/src/main/kotlin/org/jetbrains/research/kotlin/mpp/inference/operators/OperatorFactory.kt#L26):)